src/Http/Features/src/PublicAPI.Shipped.txt

```diff
@@ -0,0 +1,2 @@
+#nullable enable
```

Comment:
Should `PublicAPI.Shipped.txt` be changing?

Reply:
Why? Nothing has shipped yet in this assembly. Did you think this was `src/Http/Http.Features/`? I wonder if we should rename the folders. That tripped me up earlier in the review too.

Reply:
Well CodeCheck agrees with you. I guess it doesn't like the file at all even if empty. Still think the folder names are confusing.

Reply:
@JunTaoLuo? This is a new assembly and the tools failed if it didn't have this file.

Reply:
> Why? Nothing has shipped yet in this assembly.

That's why I was asking why it was changing

Reply:
Note Code Check is complaining about different files because they were deleted:
```
error : Modified API baseline files:
error : src/Http/Http.Features/src/PublicAPI/net6.0/PublicAPI.Shipped.txt
error : src/Http/Http.Features/src/PublicAPI/netstandard2.0/PublicAPI.Shipped.txt
```

Reply:
Once you get the CI to green ignoring the Code Check job, let @dotnet/aspnet-build know and we'll override the required checks to squishy-merge your change.

Reply:
From build perspective, the changes here make sense and simplify our PublicAPI files.